### ISSUE TITLE
Call out to, or introduce uproot-browser?

### ISSUE THREAD

**Issue by klieret** [hsf-training/hsf-training-scikit-hep-webpage#9]

**Comment by amangoel185** [comment]
Sounds like a good idea! Especially since we explore the `.root` files in the lesson. `uproot-browser` would be a nice addition to the lesson flow. Further, it shouldn't take up a lot of time introducing it to the participants.

**Comment by klieret** [comment]
If we add it to the binder requirements, they can just use the jupyter lab terminal to try it, right?  If we perhaps also include a data file or a softlink to the example data

**Comment by amangoel185** [comment]
Yep, it should be directly accessible via the Jupyter terminal! And even via the cells by using `!` but not sure how well it'll render.

**Comment by klieret** [comment]
With `!` you cannot use interactive programs, I think (because it doesn't transmit keystrokes) See [SO](https://stackoverflow.com/questions/44604027/running-interactive-command-line-code-from-jupyter-notebook).

**Comment by amangoel185** [comment]
Ah! yes. Forgot to mention that. The TUI wouldn't work, but other CLI commands like `tree` and `plot` do!
